src/main/java/org/jitsi/jicofo/OctoChannelAllocator.java

```diff
@@ -89,11 +91,12 @@ public OctoChannelAllocator(
         List<ContentPacketExtension> contents = new ArrayList<>();
         contents.add(
             jingleOfferFactory.createAudioContent(
-                    !useIce, useDtls, config.stereoEnabled()));
+                    !useIce, useDtls, config.stereoEnabled(),
```

Comment:
The content for audio uses the config, but the config for video is always false?

Reply:
@gpolitis is this now fixed ? I don't see such issue in the code anymore, am I missing something ?

Reply:
yes, now it's fixed